hugegraph-core/src/main/java/com/baidu/hugegraph/backend/cache/CacheNotifier.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.backend.cache;
+
+import com.baidu.hugegraph.backend.id.Id;
+import com.baidu.hugegraph.type.HugeType;
+
+public interface CacheNotifier extends AutoCloseable {
+
+    public void invalid(HugeType type, Id id);
+
+    public void invalid2(HugeType type, Object[] ids);
```

Comment:
any friendship name?

Reply:
sofa-rpc don't allowed to use the same method name in a service, may be names like invalidBatch() is more friendship, here just to keep the name short.